packages/utilities/web-vitals/README.md

```diff
@@ -0,0 +1,75 @@
+
+## Description
+
+The `WebVitals` utility provides a configurable hook which will collect device capability metrics (effective network type, device CPU count, and device memory) and Web Vitals metrics (Cumulative Layout Shift, Time To First Byte, etc) for each page view, and then send those metrics to a reporting endpoint when the user leaves the page.
+
+## Installation
+
+```jsx
+npm install @bbc/web-vitals --save
+```
+
+## Props
+
+| Argument          | Type     | Required | Default     | Example             |
+| ----------------- | -------- | -------- | ----------- | ------------------- |
+| enabled           | Boolean  | No       | false       | `{ enabled: true }` |
+| reportingEndpoint | String   | Yes      | N/A         | `{ reportingEndpoint: 'https://url.to.report.to/analytics' }` |
+| loggerCallback    | Function | No       | `() => {}`  | `{ loggerCallback: (error) => console.error(error) }` |
+| sampleRate        | Integer  | No       | 100         | `{ sampleRate: 5 }` |
+| reportParams      | Object   | No       | `undefined` | `{ reportParams: { pageType: 'STY' }` |
+
+## Usage
+
+The hook can be configured in a number of ways:
+
+* `enabled` - setting this to `true` will enable sending these metrics to the reporting endpoint. This is so that the reporting functionality can be turned on only when users have agreed to the appropriate privacy and tracking settings where appropriate.
+* `reportingEndpoint` - the endpoint you wish to send your metrics to
+* `loggerCallback` - a function that will be called whenever an attempt to send the metrics fails.
+* `sampleRate` - providing a number between 1-100 will set the percentage of sampling, to prevent overloading the backend reporting server. e.g. a `sampleRate` of `5` will lead to a 5% change that a given page view's metrics will be sent to the reporting server.
```

Comment:
Tiny typo - `chance` instead of `change`.

Or perhaps a wee reword - 
```
e.g. a `sampleRate` of `5` will mean that metrics for 5% of requests will be sent to the reporting server.
```